scripts/extract-rao-runner-load-from-logs.py

```diff
@@ -0,0 +1,78 @@
+#
+import argparse
+import re
+import plotly.express as px
+import pandas as pd
+import logging
+
+HELP_MESSAGE = "extract-rao-runner-load-from-logs.py -i <logs-file>"
+TIMESTAMP_REGEX = "\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}.\d{3}"
+RAO_REQUEST_RECEIVED = "RAO request received"
+RAO_REQUEST_SENT = "RAO response sent"
```

Comment:
I think it could be better to rename this variable RAO_RESPONSE_SENT instead of RAO_REQUEST_SENT

Reply:
Done